backend/src/utils/memberUtil.ts

```diff
@@ -13,6 +15,16 @@ export const filterImagesResponse = (
       fileName: image.fileName.slice(image.fileName.indexOf('/') + 1)
     }));
 
+export const getMemberFromDocumentReference = async (
+  docRef: firestore.DocumentReference
+): Promise<IdolMember> => {
+  const snapshot = await docRef.get();
+  if (!snapshot.exists) {
+    return archivedMembersByEmail[docRef.id];
```

Comment:
Looks like archived members are manually being put in. This means if someone is removed from Idol, they are not automatically archived which could cause future bugs. Should we look into automating archived members?

Reply:
Actually I'm not sure if it's possible to do this in any automated way with the current way archiving works. Archived members are just stored in a JSON file and I don't believe it'd be possible to write to this file since we have a serverless backend. 

I also think it might be unnecessary -- we shouldn't be deleting current members for no reason and as long as archiving happens at the end of every semester/beginning of a new semester then this shouldn't be an issue.